src/st/no0/e_elevator.c

```diff
@@ -0,0 +1,264 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "no0.h"
+
+s16 func_us_801C1F98(Primitive* prim, s16 arg1) {
```

Comment:
I see these dups

```
| 1.00 | true     | func_8019040C                       | asm/us/st/cen/matchings/F890/func_8019040C.s 
| 1.00 | true     | func_us_801C1F98                    | asm/us/st/no0/matchings/41F98/func_us_801C1F98.s 


| 1.00 | true     | EntityUnkId1B                       | asm/us/st/cen/matchings/F890/EntityUnkId1B.s 
| 1.00 | true     | func_us_801C26B8                    | asm/us/st/no0/matchings/41F98/func_us_801C26B8.s 
```

I'm thinking the other functions might also be the same as the CEN elevator but not similar enough to end up in the report? Didn't investigate in detail. You don't need to dedupe these right now, just wanted to bring it up

Reply:
They are reasonably close, I used them as a reference for this decomp but quite a few differences that might make it hard to dedupe